.spi.yml

```
@@ -21,3 +21,5 @@ builder:
       - SwiftSyntaxMacros
       - SwiftSyntaxMacroExpansion
       - SwiftSyntaxMacrosTestSupport
+      custom_documentation_parameters: [--experimental-skip-synthesized-symbols]
```

Should this use a single dash? When I set it as `OTHER_DOCC_FLAGS`, I need to use a single dash, not two.

```suggestion
      custom_documentation_parameters: [-experimental-skip-synthesized-symbols]
```

---

In my testing this took two dashes, because it's a parameter on `swift package generate-docs` (we generate the docs with SwiftPM). This might be the case if you generate with docs with `xcodebuild` on iOS?

Either way, I'll need to verify it runs with 5.8 so I'll double check tomorrow and will report back!

---

OK, if it works with two dashes, I’m all in favor for using that version. 👍🏽